test/end_to_end/general_setup/general.ts

```diff
@@ -410,30 +416,37 @@ export class ScenarioRunner {
         throw new Error("Unrecognized rate oracle");
       }
     }
+
+    return [marginEngine, vamm, fcm];
   }
 
-  async configureIRS() {
+  async configureIRS(
+    marginEngine: IMarginEngine,
+    vamm: IVAMM
+  ): Promise<[IMarginEngine, IVAMM]> {
```

Comment:
any need of returning `[IMarginEngine, IVAMM]`?

Reply:
To reuse when deploying a 2nd pool based on the same setup (same factory, token etc)

Reply:
Got it. But I am just wondering if we need to return the result, because in all cases there will be `[a,b] = configureIRS(a,b)`.

Reply:
LGTM overall! Glad to see that you've added support for multiple pools in the testing framework. Keep it up!

Reply:
@0xZenus happy to merge?